src/functions/helper_financial.ts

```diff
@@ -31,3 +31,14 @@ export function checkDayCountConvention(dayCountConvention: number) {
     )
   );
 }
+
+export function checkRedemptionPositive(redemption: number) {
```

Comment:
```checkRedemptionPositive``` --> ```checkRedemptionStrictlyPositive```

Reply:
Changed to `assertRedemptionPositive`.  All the functions "strictly positive" are named `assert...Positive`, and the oters are `assert...PositiveOrZero`